### ISSUE TITLE
Add UNIX socket support

### ISSUE THREAD

**Issue by rybakit** [tarantool/tarantool-php#85]
E.g.:

```php
$t = new Tarantool('unix:///tmp/tnt.sock');
```
Socket URI format: http://php.net/manual/en/transports.unix.php

**Comment by bigbes** [comment]
It'll be added in PHP7 branch, since it needs refactoring of API.

**Comment by Totktonada** [comment]
See https://github.com/bigbes/tarantool-php/commits/php7-v2

**Comment by rybakit** [comment]
> See https://github.com/bigbes/tarantool-php/commits/php7-v2

@Totktonada Is this link correct? I don't see anything related to unix sockets there.

**Comment by Totktonada** [comment]
Yep. See https://github.com/bigbes/tarantool-php/commit/35daa0ad620967d4f6d9ff76a97a0fdddd803cad and https://github.com/bigbes/tarantool-php/commit/bf6ef5742c0cb60fba0f85bff85bf64795a11a0b.
